frontend/crd/src/app/milestones-page/milestones/milestone.service.ts

```diff
@@ -1,26 +1,46 @@
 import { Injectable } from '@angular/core';
-import {HttpClient} from "@angular/common/http";
-import {Milestone, MilestoneJSON} from "../../../domain/Milestone";
-import {map, Observable} from "rxjs";
+import { HttpClient } from "@angular/common/http";
```

Comment:
If we setup ESLint we can automatically fix stuff like this

Reply:
until then i'll keep doing it manually